scripts/setup_controller.sh

```diff
@@ -104,18 +104,24 @@ set -ex
         wait_for_apt_lock
         apt-get -y --force-yes install glusterfs-client                 >> /tmp/apt3.log
     elif [ "$fileServerType" = "azurefiles" ]; then
-        # install azure cli & setup container
-        AZ_REPO=$(lsb_release -cs)
-        wget -O - https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor | sudo tee /usr/share/keyrings/microsoft-archive-keyring.gpg
-        echo "deb [arch=amd64 signed-by=/usr/share/keyrings/microsoft-archive-keyring.gpg] https://packages.microsoft.com/repos/azure-cli/ $AZ_REPO main" |  tee /etc/apt/sources.list.d/azure-cli.list
-        wait_for_apt_lock
-        sudo apt-get -y install apt-transport-https >> /tmp/apt3.log
-        wait_for_apt_lock
-        sudo apt-get -y update > /dev/null
-        wait_for_apt_lock
-        sudo apt-get -y install azure-cli >> /tmp/apt3.log
-        wait_for_apt_lock
-        apt-get -y --force-yes install cifs-utils >> /tmp/apt3.log
+        if [ "$storageAccountType" = "Premium_LRS" ]; then
```

Comment:
== instead of =

Reply:
= is the correct operator here to compare two strings